src/styles/createGenerateClassName.js

```diff
@@ -10,7 +10,7 @@ import type { Rule, generateClassName } from 'jss/lib/types';
 //
 // It's an improved version of
 // https://github.com/cssinjs/jss/blob/4e6a05dd3f7b6572fdd3ab216861d9e446c20331/src/utils/createGenerateClassName.js
-export default function createGenerateClassName(): generateClassName {
+export default function createGenerateClassName(prefix: string = "c"): generateClassName {
```

Comment:
@thangngoc89 Not too far off, but there are a few settings applied in .eslintrc.js

```
    'prettier/prettier': [
      'error',
      {
        singleQuote: true,
        printWidth: 100,
        trailingComma: 'all',
      },
    ],
```

Thanks for contributing!